package.json

```diff
@@ -620,4 +628,4 @@
     "trailingComma": "none",
     "bracketSpacing": true
   }
-}
+}
```

Comment:
Code style nitpick: empty line in the end of the file disappeared.

Reply:
This is behavior of VSCode Json editor: https://github.com/microsoft/vscode/issues/23260